zephyr/wrapper.c

```diff
@@ -336,4 +336,5 @@ void k_sys_fatal_error_handler(unsigned int reason,
 	ipc_send_panic_notification();
 
 	LOG_ERR("Halting system");
+	arch_system_halt(reason);
```

Comment:
possible set to debug only?

Reply:
maybe it is used for waiting GDB attach

Reply:
We are overriding the Zephyr panic handlers here, so we do need to halt the system here, whether it's debug or not.